vin/app/models/shipment.rb

```diff
@@ -1,47 +0,0 @@
-#enconding: utf-8
-
-class Shipment < ActiveRecord::Base
-  # Attributes
-  # attr_accessible :date, :status
-
-  # Associations
-  belongs_to :subscriber
-  has_one :package
-
-  # Validations
-  # validates :status, format: { with: /\Adelivered\Z|\Acancelled\Z|\Ascheduled\Z/ }
-  validates :status, inclusion: { in: %w(delivered cancelled scheduled) }
-
-  # Class methods
-  def self.selection_month
-    @relation.map { |s| s.selection_month }
-  end
-
-  # Methods
-  def selection_month
-    selection_month = self.date.strftime("%b/%Y")
-    {
-      id: self.id,
-      selection_month: selection_month,
-      status: self.status
-    }
-  end
-
-  def show
```

Comment:
All the "selection_month" stuff is still pending... Thanks for the gem btw, seems neat